src/server/api/routers/feedback.ts

```diff
@@ -0,0 +1,64 @@
+import { z } from 'zod'
+import * as schema from '~/server/db/migration/schema'
+
+import { createTRPCRouter, integrationProcedure } from '~/server/api/trpc'
+import { db } from '~/server/db'
+import { FeedbackTypeEnum } from '~/model/feedback'
+import { uuidv7 } from 'uuidv7'
+import { asc, eq, sql } from 'drizzle-orm'
+
+export const createFeedback = createTRPCRouter({
+  createRating: integrationProcedure
+    .input(
+      z.object({
+        threadId: z.string(),
+        chatId: z.string(),
+        feedbackType: z.nativeEnum(FeedbackTypeEnum),
+        notes: z.string().optional(),
+      }),
+    )
+    .mutation(async ({ input, ctx }) => {
+      const feedbackId = uuidv7()
+      await db
+        .insert(schema.chat_feedback)
+        .values({
+          id: feedbackId,
+          threadId: input.threadId,
+          chatId: input.chatId,
+          typeId: input.feedbackType,
+          notes: input.notes,
+          createdAt: new Date(),
+        })
+        .returning()
+    }),
+
+  getList: integrationProcedure
```

Comment:
We shouldn't allow integrated users to get list of feedbacks
Should be admin only. Let's use protectedProcedure